tasks/jira/src/main/java/com/walmartlabs/concord/plugins/jira/JiraTaskCommon.java

```diff
@@ -508,4 +490,12 @@ private String configureStatus(String projectKey, String issueType, String issue
         return jqlQuery;
     }
 
+    JiraHttpClient getClient(TaskParams in) {
+        return JiraHttpClientFactory.create(in);
```

Comment:
If the factory method is static do we really need the factory class? Looks there are no other uses of JiraHttpClientFactory and the client can be created right here?

Reply:
You're right. I was considering making it an `@Inject`able instance...not necessary though.